openpower/configs/witherspoon_defconfig

```diff
@@ -72,3 +72,4 @@ BR2_OCC_GPU_BIN_BUILD=y
 BR2_CAPP_UCODE_BIN_FILENAME="cappucode.bin"
 BR2_PACKAGE_LOADKEYS=y
 BR2_IMA_CATALOG_DTS="POWER9"
+BR2_OPENPOWER_SIGNED_SECURITY_VERSION="0"
```

Comment:
Maybe you should provide that as default on the Config.in file and not here?

Reply:
No, this is exactly where we want it.  We only want to affect one platform at this time - Witherspoon.  This will allow other platforms to use this when they want to.